SampleApp/package.json

```diff
@@ -22,7 +22,7 @@
     "react-native-moengage-geofence": "^4.0.0",
     "react-native-moengage-inbox": "^3.0.0",
     "react-native-render-html": "^6.3.4",
-    "react-native-safe-area-context": "^4.9.0",
+    "react-native-safe-area-context": "3.3.2",
```

Comment:
This change required as react-native-safe-area-context is in experimental phase for new arch and build is failing for Android with new version